HIP/hip-513.md

```diff
@@ -0,0 +1,293 @@
+---
+hip: 513
+title: Smart Contract Traceability Extension
+author: Stoyan Panayotov <stoyan.panayotov@limechain.tech>, Mustafa Uzun <mustafa.uzun@limechain.tech>
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Draft
+last-call-date-time:
+created: 2022-06-08
+discussions-to:
+updated:
+requires: 206, 435, 260
+---
```

Comment:
This replaces HIP-260 so we should say so in the metadata.

```suggestion
replaces: 260
---
```

When it goes to accepted we will also need to update HIP-260 with "superseded-by" metadata